CONTRIBUTING.md

```diff
@@ -95,6 +96,98 @@ have more control over the process and debug any issues.
 1. Run `rush change` to generate changelog. You will need to follow the interactive prompts.
    You can edit the added files later. If you don't add the right entries, the CI build will fail.
 
+# How to add and roll out new linter rules
```

Comment:
>How to add and roll out [](http://example.com/codeflow?start=2&length=23)

Please open a PBI for yourself for next sprint to apply this criteria for the set of rules currently running in staging and do one round of promotion by end of June using this criteria

Reply:
Opened [24274064](https://msazure.visualstudio.com/One/_workitems/edit/24274064)